src/phases/tpch/normalized/Q1.yml

```diff
@@ -0,0 +1,55 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/product-query"
+Description: |
+  Run TPC-H query 1 against the normalized schema. Using an 'executionStats' explain causes each command to run its execution plan until no
+  documents remain, which ensures that the query executes in its entirety.
```

Comment:
It would be quite useful to add as a comment the equivalent SQL query.

Reply:
Or maybe attach the link to the TPC-H Specification as you did in the jstest?  (http://tpc.org/tpc_documents_current_versions/pdf/tpc-h_v3.0.0.pdf)

Reply:
I added a link to the TPC-H spec to each query file.